receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -9,3 +9,47 @@ status:
   distributions: [liatrio]
   codeowners:
     active: [adrielp, astencel-sumo]
+
+# this might need to be unique per sub receiver implementation
+resource_attributes:
+  organization.name:
+      enabled: true
+      description: Git Organization or Project Name
+      type: string
+  git.vendor.name:
+      enabled: true
+      # github, gitlab, bitbucket, gittea
+      description: The name of the Git vendor/provider (ie. GitHub / GitLab)
+      type: string
```

Comment:
I don't see those being set on the scraper.

Reply:
Thanks for calling this out. Just added those resource attributes via the ResourceBuilder.